src/core/analytics.js

```diff
@@ -342,6 +354,7 @@ class Analytics {
               // logger.debug(msg);
               this.errorReporting.leaveBreadcrumb(msg);
               intgInstance = new intMod[modName](intg.config, self, intg.destinationInfo);
+              this.validateConfig(intgInstance.getRequiredConfigs(), intg.config, intg.name);
```

Comment:
Not all destinations have implemented the `getRequiredConfigs` method. So, initialization will fail for them right way.
We should first check if that method is defined.

Reply:
Moreover, it'll also not work if the user has locked the integrations version.